notify/notify.go

```diff
@@ -785,7 +785,12 @@ func (n SetNotifiesStage) Exec(ctx context.Context, l log.Logger, alerts ...*typ
 		return ctx, nil, errors.New("resolved alerts missing")
 	}
 
-	return ctx, alerts, n.nflog.Log(n.recv, gkey, firing, resolved)
+	var expiry time.Duration
+	if n, ok := RepeatInterval(ctx); ok {
```

Comment:
The repeat interval should always be present in the context? At least  `*DedupStage.Exec()` returns an error if this is the case. I suggest that we do the same here.

Reply:
Done!